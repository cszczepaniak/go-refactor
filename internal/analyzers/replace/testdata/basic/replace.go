package basic

func ReplaceMe(a, b, c int) bool {
	return false
}

func foobar() {
	r := ReplaceMe(1, 2, 3) // want "replace the function call"
	_ = r
}
