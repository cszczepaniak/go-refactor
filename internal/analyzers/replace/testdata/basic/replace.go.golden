package basic

func ReplaceMe(a, b, c int) bool {
	return false
}

func foobar() {
	r := ExampleReplacement // want "replace the function call"
	_ = r
}
